### PR TITLE
[6.3] fix katello agent not started automatically anymore

### PR DESCRIPTION
**Pull request by ldjebran**
portion of test log: http://pastebin.test.redhat.com/495623
also we can notice by this log the bug affection: https://bugzilla.redhat.com/show_bug.cgi?id=1457977

### CODE REVIEW

File: robottelo/vm.py [hunk 1]
```diff
@@ -269,10 +268,9 @@ def install_katello_agent(self):
         result = self.run('rpm -q katello-agent')
         if result.return_code != 0:
             raise VirtualMachineError('Failed to install katello-agent')
-        if bz_bug_is_open('1431747'):
-            gofer_start = self.run('service goferd start')
-            if gofer_start.return_code != 0:
-                raise VirtualMachineError('Failed to start katello-agent')
+        gofer_start = self.run('service goferd start')
```
Comment (oshtaier) [inline robottelo/vm.py]: Not sure that after I read both bugs description I see confirmation that goferd should not start right away after installation is finished
Yes, there is a chance that you should have some delay, but it is wrong approach to start service manually
Correct me if I am wrong, till that moment NACK on that fix

**Comment by sghai** [comment]
@ldjebran : Just a note: I heard recently from dev's that in 6.3 we are not going to start goferd automatically. Plan is to use the REx for package install/update functionality.. And errata applicability would be now generated through yum plugin in 6.3.

More info on internal QA list.

File: robottelo/vm.py (hunk 1 above)
Comment (ldjebran) [inline robottelo/vm.py]: @oshtaier we had yesterday a discussion on IRC and Eric Helm confirmed that Katello agent is not expected to auto-start any more on 6.3

**Comment by oshtaier** [comment]
@sghai thanks for the information

**Comment by ldjebran** [comment]
make it do not merge, this need more investigations

**Comment by ldjebran** [comment]
investigation done: goferd not starting automatically (and will not), new feature not ready for the moment.

**Comment by ldjebran** [comment]
@oshtaier @sghai to make a recap:

* we have a new functionality gofer-less
* goferd still maintained
* goferd not started at install stage

**Comment by ldjebran** [comment]
@oshtaier is your NACK still relevant ?

**Comment by ldjebran** [comment]
put to do not merge, wait for confirmation message about the startup functionality

**Comment by ldjebran** [comment]
@omaciel as per the mail list I understood that for 6.3 no change for gofer and for goferd, what do you think should we close this PR waiting for the build if goferd fail to start at install time file a bug about that

**Comment by omaciel** [comment]
@ldjebran makes a ton of sense to me :)

**Comment by ldjebran** [comment]
closing this PR as currently the changes are discarded

**Pull request closed without merging**